redi-connect-admin/src/App.js

```diff
@@ -756,12 +672,12 @@ const RedProfileEdit = (props) => (
 );
 
 const CategoriesInput = (props) => {
-  const categories = categoriesByLocation[props.record.rediLocation];
+  const categories = categoriesFlat;
   return (
     <SelectArrayInput
       {...props}
       source="categories"
-      label={`Categories (ONLY for ${props.record.rediLocation})  `}
+      label={`Categories`}
```

Comment:
Perhaps we dont need the ` now that we dont have any js value concatenated in the string?
```suggestion
      label="Categories"
```

Reply:
Definitely